src/doc/src/reference/unstable.md

```diff
@@ -338,26 +338,26 @@ This was stabilized in 1.79 in [#13608](https://github.com/rust-lang/cargo/pull/
 ### MSRV-aware resolver
 
 `-Zmsrv-policy` allows access to an MSRV-aware resolver which can be enabled with:
-- `resolver.something-like-precedence` config field
+- `resolver.incompatible-rust-versions` config field
 - `workspace.resolver = "3"` / `package.resolver = "3"`
 - `package.edition = "2024"` (only in workspace root)
 
 The resolver will prefer dependencies with a `package.rust-version` that is the same or older than your project's MSRV.
 Your project's MSRV is determined by taking the lowest `package.rust-version` set among your workspace members.
 If there is none set, your toolchain version will be used with the intent to pick up the version from rustup's `rust-toolchain.toml`, if present.
```

Comment:
Slightly off-topic, but we may not want to expose the implementation detail of rustup.

```suggestion
If there is none set, your toolchain version will be used. The toolchain version is usually determined by `rustup` or the system package manager. 
```

Reply:
This section isn't as important long-term as this is just describing the unstable feature.

However, in describing it this way, I'm trying to accomplish two things
- Emphasize the motivating use case for this functionality
- Help rustup users discover this by using terms they would expect

I am making a slight tweak to it so hopefully that will help.